test/e2e/webdriver/driver.js

```diff
@@ -359,6 +359,7 @@ class Driver {
     const ignoredErrorMessages = [
       // Third-party Favicon 404s show up as errors
       'favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found)',
+      'Failed to load resource: the server responded with a status of 429',
```

Comment:
Nit: A comment to explain what this error means might be helpful here

```suggestion
      // Sentry rate limiting
      'Failed to load resource: the server responded with a status of 429',
```